docs/develop/ethereum-to-ton/difference-of-blockchains.md

```diff
@@ -0,0 +1,56 @@
+
+Now let's understand what is actor model? Actor model is an approach to parallel and distributed computing where the main element is an actor - an independent executable block of code. Originally developed for cluster computing, this model is widely used in micro-server architectures to meet the needs of modern distributed systems due to its ability to scale, parallelism and fault tolerance. Actors receive and process messages, depending on the logic of the message, respond by accepting local changes or performing actions in response, can create other actors or send messages onward. They are thread-safe and reentrant, eliminating the need for locks and simplifying parallel processing of tasks. This model is ideal for building scalable and reliable server solutions, providing efficient concurrent access control and support for both synchronous and asynchronous messaging. 
+
+In TON, everything is represented by smart contracts, which can also be called actor in the context of actor model. A smart contract is an object with properties such as address, code, data and balance. It has the ability to store data and behaves according to instructions received from other smart contracts. After a contract receives a message and processes it by executing its code in the TVM, various scenarios can occur:
+- The contract changes its properties `code, data, balance`
+- The contract optionally generates an outgoing message
+- The contract goes into standby mode until the following event occurs
+
+The result of scripts is always the creation of a transaction.  The transactions themselves are asynchronous, meaning that the system can continue processing other transactions while waiting for past transactions to complete. This provides more flexibility when processing complex transactions. Sometimes a single transaction may require multiple smart contract calls to be executed in a specific sequence. Because these calls are asynchronous, developers can more easily design and implement complex transaction flows that may involve multiple concurrent operations. A developer coming from Ethereum needs to realize that smart contracts in the TON blockchain can only communicate with each other by sending asynchronous messages, which means that if there is a need to request data from another contract and an immediate response is required, this will not be possible. Instead `get methods` must be called by clients outside the network, much like a wallet in Ethereum uses RPC nodes such as Infura to request smart contract states.
+
```

Comment:
"Instead `get methods` must be called by clients outside the network" - it is worth mentioning that it is an important limitation. Flesh loans are impossible which is crucial in DEFI. Also, Oracles use case is more complicated